lib/hammer_cli_foreman/audit.rb

```diff
@@ -61,6 +61,7 @@ def extend_data(audit)
           field :old, _("Old"), Fields::LongText, :hide_blank => true
           field :new, _("New"), Fields::LongText, :hide_blank => true
         end
+        field :request_uuid, _("Request UUID")
```

Comment:
I think this field can be moved in the `ListCommand` output definition, but with field sets specified like so:
```ruby
field :request_uuid, _("Request UUID"), Fields::Field, sets: ['ALL']
```
to be shown only by explicit request with `--fields 'ALL'` option.